geoengine/permissions.py

```diff
@@ -0,0 +1,205 @@
+'''
+A wrapper for the GeoEngine permissions API.
+'''
+
+from __future__ import annotations
+
+from typing import Dict
+from uuid import UUID
+import json
+
+import requests as req
+
+from geoengine.api import Permission, ResourceId
```

Comment:
please don't re-export types from api. It should be clear that they are only API types. We might generate them one day and then everything breaks.

Reply:
cb77f6e92122f0cc346dbc4ab2320a44a0ceea55